_posts/2020-11-24-Multiple-Audio-Interfaces.md

```diff
@@ -19,7 +19,7 @@ Under Windows, the best option to run multiple interfaces through Jamulus at the
 **Steps:**
 1. [Install and configure JACK on Windows](https://jackaudio.org/faq/jack_on_windows.html) - Make sure you select your best audio interface in the configuration steps, also add `-r 48000 -p <desired frame buffer size>` to ensure JACK is outputting audio in the required sample rate for Jamulus (48.000Hz). If your frame buffer size is too low for your hardware, you'll need to change it to a higher value;
 2. Install and open Voicemeeter;
-3. Open configured Jack PortAudio shortcut, Jack Control and Jamulus software;
+3. Open configured JACK PortAudio shortcut, Jack Control and Jamulus software;
 4. On Voicemeeter, go to HARDWARE OUT and on A1 dropdown list, select JackRouter. On Hardware Input 1 select your computer microphone. You can make your computer microphone have the lowest latency possible by going into _Menu > System Settings/Options..._ and enabling _WDM Input Exclusive Mode_ (if you experience crackling noises while speaking, disable this) and changing the _Engine Mode_ to _Swift_;
 5. Make sure the Hardware Input where you selected your microphone is sending sound through A1 by enabling the corresponding button next to the fader;
 6. On Jamulus, under Settings, select JackRouter as your Soundcard Device;
```

Comment:
To discuss: Soundcard, Sound card, Interface etc. There's already an issue about that.

Reply:
Correct, this is another consistency PR, please @ann0see keep it on topic, this one is jack/Jack/JACK.